Our projects page completed

jayp822 opened the pull request:
Created about/projects directory page. Didn't have title, description, image, and link so I just used temp data. Made a more general carousel component that can be used by others in other parts of the website as well. 

https://github.com/user-attachments/assets/db672f3f-6d61-406d-899e-cd0ae82bfbc1

justinquinnb reviewed (commented):
Temp data is fine and the page looks good. We'll get real data in there as we begin the project this year. For the Carousel Component, can we add some spacing between carousel items and adjust their sizing to match the Figma? The card at the center should be bigger than the next and previous items in the carousel, visible on the left and right. When the carousel changes to the next card, they should slide but also grow/shrink appropriately. If possible, the carousel component should accept 1 to an arbitrary number of items, not sure if this functionality is already implemented.

Otherwise great work yet again, grateful for all your help in such a short time frame!

jayp822 commented:
@justinquinnb yeah I can work on it just realized there is no background 😅.

justinquinnb commented:
Nw! Really appreciate your contributions, your help hasn't gone unnoticed 😀

nivedhanatarajan7 commented:
The Projects page looks great! Since we will begin working on the first DevDogs project (Optimized Schedule Builder) this year, I think we can comment the section related to past projects and just display the current project section.

jayp822 commented:
Removed Stock images used in old home carousel and updated our projects a carousel to look more like the one in figma. Currently the past projects carousel works but is commented out noted by @nivedhanatarajan7 .

https://github.com/user-attachments/assets/bd154845-8a46-4884-a65d-c42fefa33762

nivedhanatarajan7 commented on src/app/about/projects/page.js:
```diff
@@ -1,9 +1,114 @@
+   </div>
+   {/* Carousel */}
+   {/* <CarouselAdvanced /> */}
+   <EmblaCarousel slides={currentProjects} options={OPTIONS} />
```
I think we will probably be working on one project as a group at DevDogs each year or semester, so I dont think the display of the current project needs to be a carousel. It can be a picture card.

jayp822 replied:
Ok, yeah I also agree that the projects page should be like a showcase for our main project for the semester so more people are on the same page about the main goal. I could then just have like learn more button or make the card clickable so it can redirect to the github repository if you want.

nivedhanatarajan7 replied:
We plan to have a page about the Schedule Builder project, and I will probably make that once the details of the project have been confirmed. For now, there is a View button in the Projects page, that you can link to the GitHub repository for the project.

nivedhanatarajan7 commented on src/app/about/projects/page.js:
```diff
@@ -1,9 +1,114 @@
+
+
+   <p className="pt-10 text-base sm:text-lg md:text-xl">
+    DevDogs is proud to consist of ## UGA students from all tech disciplines
```
This section would contain the description of the project, so we can keep the text here as a placeholder while we get the project description ready. The number of students in DevDogs is something that can change, so I think we can just say "...consist of UGA student..." Stating "from all tech disciplines" seems like a strong statement, so I think we can reword this to be "...from many tech disciplines..." Thank you!

jayp822 replied:
I would be pretty cool if we had a live counter from our discord that could send the data to the webpage and update it in the background whenever someone renew joins. Could be a cool addition for the future

nivedhanatarajan7 commented on src/app/about/projects/page.js:
```diff
@@ -1,9 +1,114 @@
+   </p>
+   <Link
+    target="_blank"
+    href="https://github.com/DevDogs-UGA"
```
The View button should probably link to the Our People page since the paragraph above is about the members of the team.

nivedhanatarajan7 commented on public/Screenshot 2024-08-17 at 4.41.35 PM.png:
Please rename this file with a meaningful name.

nivedhanatarajan7 reviewed (changes requested):
The Project page looks good. I gave some comments regarding styling and content. It would also be nice if you could add a comment in the various carousel components since there are different components with probably varying purposes.

jayp822 commented:
@nivedhanatarajan7 should this button redirect to "/about/projects" or stay as the discord link?
![suggestion](https://github.com/user-attachments/assets/419b3a40-a130-47e3-9eda-cffa9092d486)

nivedhanatarajan7 commented:
The All Projects button should link to the Projects page. Thank you!

jayp822 commented:
Completed edits with this new addition
https://github.com/user-attachments/assets/a64cc531-bb1f-49c5-b4ab-a78d8e33d8cc
